frontend/src/Components/Elements/Table/TableModal.styled.js

```diff
@@ -0,0 +1,104 @@
+import styled from 'styled-components';
+import { phoneOnly, smallerThanDesktop, smallerThanTabletLandscape } from 'styles/breakpoints';
+import { P } from 'styles/StyledComponents/Typography';
+import FJIcon from 'img/icons/Icon';
+
+export const TableModal = styled.div`
+  z-index: 21;
+  background: ${(p) => p.theme.colors.white};
+
+  padding: 2.5em;
+  width: 100%;
+  max-width: 1000px;
+  height: 100vh;
+
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  overflow-y: scroll;
+
+  @media (${smallerThanDesktop}) {
+    max-width: 80%;
+  }
+
+  @media (${smallerThanTabletLandscape}) {
+    width: 95vw;
+    height: 95vh;
```

Comment:
The modal expands past the window on mobile view for me, but not sure we have time to fix it.